### PR TITLE
[MOL-142] all menu items tooltips/modals

### PR DESCRIPTION
**Pull request by mev1482**
### Description
Adding tool tips/Modals to the rest of the menu items

### Checklist
- [X] Code compiles correctly
- [X] Created tests which fail without the change (if possible)
- [X] All tests passing
- [X] Extended the README / documentation, if necessary

### CODE REVIEW

File: src/builder/form/TestOptions.js [hunk 1]
```diff
@@ -46,17 +67,10 @@ function TestOptions(props) {
           onChange={(e) => handleChange(e, 0)}
         /> : null
       }
-      <UploadFile
```
Comment (mag8224) [inline src/builder/form/TestOptions.js:49]: What UploadFile component is this removing?

Comment (mev1482) [reply src/builder/form/TestOptions.js:49]: This is the duplicate button that was in the maker object. I removed it and tested, I can re-add it and take care of it in another story if you think the two should be separate.

Comment (mag8224) [reply src/builder/form/TestOptions.js:49]: I think it's good to remove here, just wanted to make sure it was the duplicate one and not one currently being used.

File: src/builder/form/BuilderForm.js [hunk 2]
```diff
@@ -20,7 +21,7 @@ const MAX_RESIDUE = 10;
 const MIN_RESIDUE = 3;
 
 function BuilderForm(props) {
-  const { classes, values, handleChange } = props;
+  const { classes, values, handleChange, helpText} = props;
```
Comment (arp6695) [inline src/builder/form/BuilderForm.js:24]: Redline. Adjust spacing on the last brace.

File: src/common/FilteredLigandResults.js [hunk 3]
```diff
@@ -159,9 +174,9 @@ FilteredLigandResults.propTypes = {
   dockHandler: propTypes.func,
   /** The handler for selecting a  */
   handleSelectedResult: propTypes.func,
-  /** An object representing a ligand selected to be viewed
```
Comment (arp6695) [inline src/common/FilteredLigandResults.js:162]: Can we include these semantic corrections in other commits/pull requests? It's not like we need to review these.

**Review by arp6695 — APPROVED** [review]
Verified.

**Review by mag8224 — APPROVED** [review]

**Review by Dennis-n — APPROVED** [review]

**Pull request merged**